src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -162,7 +162,96 @@ internal class DaprClientGrpc : DaprClient
             }
         }
 
+        /// <inheritdoc/>
+        public override Task<BulkPublishResponse> BulkPublishEventAsync<TValue>(
+            string pubsubName,
+            string topicName,
+            IReadOnlyList<TValue> events,
+            string contentType,
+            CancellationToken cancellationToken = default)
+        {
+            return BulkPublishEventAsync(pubsubName, topicName, events, contentType, null, cancellationToken);
```

Comment:
Passing an empty dictionary here removes a null check below.

Reply:
Got rid of that overload.